app/styles/themes/gothamist/_theme.overrides.scss

```diff
@@ -268,13 +365,16 @@ blockquote {
 .c-lead-gallery__thumbs-thumb.is-active {
   border-color: rgb(var(--color-reddish-orange));
 }
+
 .c-lead-gallery__thumbs-thumb-text:hover {
   background-color: rgb(var(-color-reddish-orange));
 }
+
 .c-lead-gallery .o-credit a:hover {
   color: rgb(var(--color-link-hover));
-  background-color: rgb(var(--color-redd  ish-orange));
+  background-color: rgb(var(--color-redd ish-orange));
```

Comment:
Not sure if it's github's diff view or if there's still a space here

Reply:
weird, fixed it!